services/121-service/src/events/interfaces/create-from-distinct-registration-pair.interface.ts

```diff
@@ -0,0 +1,7 @@
+import { RegistrationIdentifiers } from '@121-service/src/events/interfaces/registration-identifiers.interface';
```

Comment:
On the topic of naming...

@RubenGeo:

Would it be better to have a singular term for this, like "RegistrationIdentity"?
As it is not a collection/set/array of things...

Reply:
resolved with rebase